install.sh

```diff
@@ -92,32 +101,42 @@ if [[ -z "$APARAVI_PARENT_OBJECT_ID" ]]; then
 fi
 }
 ###### end of required switches checking ###### 
-
+NODE_ANSIBLE_SKIP_TAGS=""
+[[ -z "$HARDENING_ADVANCED_ADD" ]]&&HARDENING_ADVANCED_ADD="no"
+[[ "$HARDENING_ADVANCED_ADD" == "yes" ]]&&HARDENING_ADVANCED_TAG=",hardening_advanced"||$HARDENING_ADVANCED_TAG=""
+[[ -z "$HARDENING_PARTITIONS_ADD" ]]&&HARDENING_PARTITIONS_ADD="no"
+[[ "$HARDENING_PARTITIONS_ADD" == "yes" ]]&&HARDENING_PARTITIONS_TAG=",hardening_partitions"||$HARDENING_PARTITIONS_TAG=""
 ###### Node profile dictionary ######
 [[ -z "$NODE_PROFILE" ]]&&NODE_PROFILE="basic"
 
     case "${NODE_PROFILE}" in
         basic)
-            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening"
+            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening${HARDENING_ADVANCED_TAG}${HARDENING_PARTITIONS_TAG}"
+            ;;
+        hardening)
+            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,hardening_advanced,wazuh_agent${HARDENING_PARTITIONS_TAG}"
             ;;
         secure)
-            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,clamav_agent,wazuh_agent"
+            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,clamav_agent,wazuh_agent${HARDENING_ADVANCED_TAG}${HARDENING_PARTITIONS_TAG}"
             ;;
         monitoring)
             check_c_switch
-            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,logs_collection,prometheus_node_exporter"
+            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,logs_collection,prometheus_node_exporter${HARDENING_ADVANCED_TAG}${HARDENING_PARTITIONS_TAG}"
             ;;
         appliance)
             check_o_switch
-            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,mysql_server,aparavi_appagent"
+            NODE_ANSIBLE_TAGS="-t os_hardening,ssh_hardening,mysql_server,aparavi_appagent${HARDENING_ADVANCED_TAG}${HARDENING_PARTITIONS_TAG}"
             ;;
         full)
             check_c_switch
             check_o_switch
             NODE_ANSIBLE_TAGS=""
+            NODE_ANSIBLE_SKIP_TAGS="--skip-tags notag"
```

Comment:
propose accumulative approach without any tags skipping, e.g.
1) basic
2) basic+adv_hardening
3) basic+adv_hardening+partitions
4) full+adv_hardening
5) full+adv_hardening+partitions

we need to set ansible tags properly to make adding new profile on demand fast and cheap

Reply:
done